default.build

```diff
@@ -319,10 +319,21 @@
 			<in>
 				<items>
 					<include name="${nuget.nupackages.dir}/*.nupkg"/>
+					<exclude name="${nuget.nupackages.dir}/*.symbols.nupkg"/>
```

Comment:
Due to having to specify the destination, symbols package needs now a different command line.